docs/sdkx_ios/sdk-configuration.mdx

```diff
@@ -49,6 +47,8 @@ Upon setting `enableLogging` to `YES`, Helpshift SDK logs will be generated in t
 | Min SDK       | v10.0.0                                           |
 | Supported by: | <code>installWithPlatformId:domain:config:</code> |
 
+</div>
```

Comment:
The below example code for install API
on android <PLATFORM_ID>, <DOMAIN>
iOS objective-c YOUR_PLATFORM_ID, YOUR_DOMAIN_NAME
iOS swit it is YOUR_PLATFORM_ID, YOUR_DOMAIN

Reply:
This is repeated in all code snippet in the page.